.gitignore

```diff
@@ -2,6 +2,7 @@
 .fileStorage/
 .vscode
 .idea
+.c9/
```

Comment:
Actually, is there a reason for that trailing slash? The other directory-wide ignores don't have that.